src/transformers/models/mask2former/convert_mask2former_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -177,7 +184,7 @@ def __call__(self, original_config: object) -> Mask2FormerFeatureExtractor:
             size=model_input.MIN_SIZE_TEST,
             max_size=model_input.MAX_SIZE_TEST,
             num_labels=model.SEM_SEG_HEAD.NUM_CLASSES,
-            ignore_index=dataset_catalog.ignore_label,
+            # ignore_index=dataset_catalog.ignore_label, #check
```

Comment:
This is probably left-over code but why is `ignore_index `commented out?

Reply:
I was getting an error actually that's why I had commented this.  

AttributeError: Attribute 'ignore_label' does not exist in the metadata of dataset 'coco_2017_val'. Available keys are dict_keys(['name', 'json_file', 'image_root', 'evaluator_type', 'thing_dataset_id_to_contiguous_id', 'thing_classes', 'thing_colors'])

Needed to debug this -> that's why added a small `check` comment